File: server/src/main/java/maker/server/Entity/style.java
package maker.server.Entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class style {
    private String outer;
    private String top;
    private String bottom;
    private String shoes;
}

File: android/app/src/main/java/com/maker/hanger/data/Style.kt
package com.maker.hanger.data

import com.google.gson.annotations.SerializedName

data class Style(
    @SerializedName("outer") var outer: String?,
    @SerializedName("top") var top: String,
    @SerializedName("bottom") var bottom: String,
    @SerializedName("shoes") var shoes: String
)